app/styles/loading.css

```diff
@@ -10,76 +9,54 @@
   float: left;
   height: 33%;
   width: 33%;
-
-  -webkit-animation: sk-cubeGridScaleDelay 1.3s infinite ease-in-out;
-  animation: sk-cubeGridScaleDelay 1.3s infinite ease-in-out;
+  animation: sk-cube-grid-scale-delay 1.3s infinite ease-in-out;
 }
 
 .sk-cube-grid .sk-cube1 {
-  -webkit-animation-delay: 0.2s;
   animation-delay: 0.2s;
 }
 
 .sk-cube-grid .sk-cube2 {
-  -webkit-animation-delay: 0.3s;
   animation-delay: 0.3s;
 }
 
 .sk-cube-grid .sk-cube3 {
-  -webkit-animation-delay: 0.4s;
   animation-delay: 0.4s;
 }
 
 .sk-cube-grid .sk-cube4 {
-  -webkit-animation-delay: 0.1s;
   animation-delay: 0.1s;
 }
 
 .sk-cube-grid .sk-cube5 {
-  -webkit-animation-delay: 0.2s;
   animation-delay: 0.2s;
 }
 
 .sk-cube-grid .sk-cube6 {
-  -webkit-animation-delay: 0.3s;
   animation-delay: 0.3s;
 }
 
 .sk-cube-grid .sk-cube7 {
-  -webkit-animation-delay: 0s;
   animation-delay: 0s;
 }
 
 .sk-cube-grid .sk-cube8 {
-  -webkit-animation-delay: 0.1s;
   animation-delay: 0.1s;
 }
 
 .sk-cube-grid .sk-cube9 {
-  -webkit-animation-delay: 0.2s;
   animation-delay: 0.2s;
 }
 
-@-webkit-keyframes sk-cubeGridScaleDelay {
-  0%, 70%, 100% {
-    -webkit-transform: scale3D(1, 1, 1);
-    transform: scale3D(1, 1, 1);
-  }
-
-  35% {
-    -webkit-transform: scale3D(0, 0, 1);
-    transform: scale3D(0, 0, 1);
-  }
-}
-
-@keyframes sk-cubeGridScaleDelay {
-  0%, 70%, 100% {
-    -webkit-transform: scale3D(1, 1, 1);
-    transform: scale3D(1, 1, 1);
+/* stylelint-disable-next-line keyframes-name-pattern */
```

Comment:
let's either fix this or leave it as a top of the file ignore so we can track it and come back to it later if we want 👍

Reply:
aah, my bad. I have fixed it but forgot to remove the disable comment